#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_plld_rename_swipl.dpatch by Chris Lamb <chris@chris-lamb.co.uk>
##
## DP: We rename the pl binary to swipl elsewhere, but plld needs to
## DP: know at compile-time.

@DPATCH@
diff -urNad pl-5.6.47~/src/plld.c pl-5.6.47/src/plld.c
--- pl-5.6.47~/src/plld.c	2007-11-19 10:01:17.000000000 +0000
+++ pl-5.6.47/src/plld.c	2007-12-09 02:35:06.000000000 +0000
@@ -77,7 +77,7 @@
 #include "pl-incl.h"
 
 #ifndef PROG_PL
-#define PROG_PL "pl"
+#define PROG_PL "swipl"
 #endif
 #define PROG_CC "cc"
 #define PROG_CXX "c++"
